shell/dialog/MoveNamespaceDialog.vue

```diff
@@ -63,8 +63,8 @@ export default {
   },
 
   methods: {
-    close(data) {
-      this.$emit('close', data);
+    close() {
+      this.$emit('close');
```

Comment:
@aalves08 I'm trying to pinpoint _why_ this emitted close with data. `MoveNamespaceDialog` is the only dialog that has this behavior. Do you recall why this is or is it an artifact of some implementation from before we refactored the modals/drodown menus?

Reply:
Because it's what triggers a callback, as per https://github.com/rancher/dashboard/pull/14187

The main aspect we lost with modals being centralised in the `PromptModal` mechanism was the ability to perform callbacks when it's being closed because the context was no longer the component itself. 

I created this store-based mechanism to handle that. This shouldn't be removed, btw